examples/blame_example.ipynb

```diff
@@ -1,310 +1,240 @@
-       "0.4857044816017151"
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "kT3m0-zoNdvW"
+      },
+      "outputs": [],
+      "source": [
+        "location_info_response = iai.location_info(location=location)\n",
+        "rendered_static_map = location_info_response.birdview_image.decode()\n",
+        "scene_plotter = iai.utils.ScenePlotter(rendered_static_map,\n",
+        "                                       location_info_response.map_fov,\n",
+        "                                       (location_info_response.map_center.x, location_info_response.map_center.y),\n",
+        "                                       location_info_response.static_actors)"
       ]
-     },
-     "execution_count": 10,
-     "metadata": {},
-     "output_type": "execute_result"
-    }
-   ],
-   "source": [
-    "blame_response.confidence_score"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 11,
-   "metadata": {},
-   "outputs": [],
-   "source": [
-    "%%capture\n",
-    "fig, ax = plt.subplots(constrained_layout=True, figsize=(15, 15))\n",
-    "gif_name = 'part-iai-drive-side-road-green.gif'\n",
-    "scene_plotter.animate_scene(output_name=gif_name, ax=ax, \n",
-    "                            start_idx=max(0, collision[1][0] - 50), end_idx=collision[1][1],\n",
-    "                            numbers=list(collision[0]), direction_vec=False, velocity_vec=False,\n",
-    "                            plot_frame_number=True)"
-   ]
-  },
-  {
-   "cell_type": "code",
-   "execution_count": 12,
-   "metadata": {},
-   "outputs": [
+    },
+    {
+      "cell_type": "code",
+      "execution_count": null,
+      "metadata": {
+        "id": "L0ufxMO8NdvX"
+      },
+      "outputs": [],
+      "source": [
+        "collisions = []\n",
+        "collision_index = None\n",
+        "is_found_collision = False\n",
+        "\n",
+        "# Simulate with `initialize`, `drive` and `light` until there are collisions.\n",
+        "for _ in range(20): #Attempt 20 simulations looking for a collision\n",
+        "    light_response = iai.light(location=location)\n",
+        "\n",
+        "    response = iai.initialize(\n",
+        "        location=location,\n",
+        "        agent_count=15,\n",
+        "        get_birdview=True,\n",
+        "        traffic_light_state_history=[light_response.traffic_lights_states]\n",
+        "    )\n",
+        "    agent_attributes = response.agent_attributes\n",
+        "    scene_plotter.initialize_recording(response.agent_states,\n",
+        "                                       agent_attributes=agent_attributes,\n",
+        "                                       traffic_light_states=light_response.traffic_lights_states)\n",
+        "\n",
+        "    agent_state_history = []\n",
+        "    traffic_light_state_history = []\n",
+        "\n",
+        "    # 20-second scene\n",
+        "    for t in range(100):\n",
```

Comment:
`t in range(100)` is not 20 seconds.

Besides, since you don't use the data after the first collision, you can stop immediately as soon as `is_found_collision`.

Reply:
Fixed.